### PR TITLE
chore: use data value directly in tests

### LINKED ISSUE #1774: FE: refactor Testcase files
# Feature request

## Please describe your feature

Hi, this issue aims to improve Frontend's Testcases (Cypress.io).

## Describe the solution you'd like

- [ ] use `data value` directly instead of referring 
- [ ] do we need to take some codes as repeated codes?
- [ ] check if testcases are standard according to Cypress docs.

### PR DESCRIPTION
**Pull request by iamayushdas**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**

- [ ] Bugfix
- [ ] New feature provided
- [x] Improve performance
- [ ] Backport patches

**What changes will this PR take into?**
improved test cases for create_and_delete_consumer.spec.js

**Related issues**

related issue #1774

**Checklist:**

- [x] Did you explain what problem does this PR solve? Or what new features have been added?
- [x] Have you added corresponding test cases?
- [ ] Have you modified the corresponding document?
- [ ] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

### CODE REVIEW

**Comment by iamayushdas** [comment]
@guoqqqi have a look

**Review by juzhiyuan — APPROVED** [review]
That's it, LGTM

**Comment by juzhiyuan** [comment]
@iamayushdas Hi, please update other test cases in other PRs. One PR with one Testcase.

**Review by juzhiyuan — APPROVED** [review]

**Comment by iamayushdas** [comment]
> @iamayushdas Hi, please update other test cases in other PRs. One PR with one Testcase.

okay sure

File: web/cypress/integration/consumer/create-consumer-with-proxy-mirror-form.spec.js [hunk 1]
```diff
@@ -17,38 +17,52 @@
...
+    pluginCard: '.ant-card',
+    drawer: '.ant-drawer-content',
+    dropdown: '.rc-virtual-list',
+    disabledSwitcher: '#disable',
+    codeMirror: '.CodeMirror',
+    notification: '.ant-notification-notice-message',
+    notificationCloseIcon: '.ant-notification-close-icon',
     host: "#host",
     alert: "[role=alert]"
   }
```
Comment (guoqqqi) [inline web/cypress/integration/consumer/create-consumer-with-proxy-mirror-form.spec.js:30]:
```suggestion
    notification: '.ant-notification-notice-message',
    notificationCloseIcon: '.ant-notification-close-icon',
     host: '#host',
     alert: '[role=alert]'
   }
```
It is best to use both single and double quotation marks

**Comment by iamayushdas** [comment]
Ok @guoqqqi

File: web/cypress/integration/consumer/create-consumer-with-proxy-mirror-form.spec.js (hunk 1 above)
Comment (juzhiyuan) [reply web/cypress/integration/consumer/create-consumer-with-proxy-mirror-form.spec.js:30]: image

Why there have spaces?

Comment (guoqqqi) [reply web/cypress/integration/consumer/create-consumer-with-proxy-mirror-form.spec.js:30]: This is my problem. I didn't see the space gap clearly when commenting on my phone……

Comment (juzhiyuan) [reply web/cypress/integration/consumer/create-consumer-with-proxy-mirror-form.spec.js:30]: ok

**Pull request closed without merging**